crates/futures/src/polyfill.js

```diff
@@ -0,0 +1,93 @@
+const helperCode = `
```

Comment:
Could this file have a comment about where it comes from?

Reply:
Oh, thanks!